src/initgeneration/initgeneration.go

```diff
@@ -150,21 +151,22 @@ func (g *InitGenerator) createSecretConfigForDynaKube(ctx context.Context, dynak
 		return nil, errors.WithStack(err)
 	}
 
-	tenantUUID, err := dynakube.TenantUUIDFromApiUrl()
+	tenantToken, err := g.getTenantToken(ctx, dynakube)
 	if err != nil {
-		return nil, errors.WithStack(err)
+		return nil, err
 	}
 
 	return &standalone.SecretConfig{
 		ApiUrl:              dynakube.Spec.APIURL,
 		ApiToken:            getAPIToken(tokens),
 		PaasToken:           getPaasToken(tokens),
+		TenantToken:         tenantToken,
+		ConnectionInfo:      dynakube.Status.OneAgent.ConnectionInfoStatus,
```

Comment:
what if this is not fast enough reconciled in the dynakube? e.g Dynakube is created and immediatly afterwards apps are started?

Reply:
the webhook would error, as it won't be able to get the `tenantToken` secret